### PR TITLE
NyanCat Lolifier

### PR DESCRIPTION
**Pull request by AlexDpy**
Hi,

Thanks for these really funny logs !

Here is my little contribution :)

### CODE REVIEW

**Comment by lebris** [comment]
:+1:

File: lib/Lolifiers/NyanCat.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+class NyanCat implements Lolifier
+{
+    public function isHandling(array $record)
+    {
+        return true;
+    }
+
+    public function lolify(array $record)
+    {
+        $message['message'] = preg_replace("/[A-Za-zÀ-ÿ0-9]+/", "nya", $record['message']);
```
Comment (devwebpeanuts) [inline lib/Lolifiers/NyanCat.php]: Be careful,you have to return `$record`. Message is replaced but other keys (like level) must be kept.

File: tests/Lolifiers/NyanCatTest.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        return array(
+            array('', ''),
+            array('   ', '   '),
+            array('nya nya nya nya', 'I am a cat'),
+            array('nya nya\'nya nya nya, nya nya, nya.', 'I don\'t care about, you know, punctuation.'),
+            array('nya nya nya nya', 'Àccéènts arê mÿ frùends'),
+            array('nya-nya ` nya +nya nya| nya !!! :(', 'I-hate ` all +those weird| symbols !!! :('),
+        );
+    }
+}
```
Comment (devwebpeanuts) [inline tests/Lolifiers/NyanCatTest.php]: Don't forget extra line at the end of files

**Comment by devwebpeanuts** [comment]
Some feedbacks :stuck_out_tongue_closed_eyes:...
Anyway, thanks for your contribution :smile: :+1:

**Comment by AlexDpy** [comment]
I just updated the code, thank you for your feedback !

**Comment by devwebpeanuts** [comment]
Thx

**Comment by AymericPlanche** [comment]
:+1:

**Pull request merged**